interfaces/web-animations.idl

```diff
@@ -92,11 +92,12 @@ dictionary ComputedEffectTiming : EffectTiming {
 
 [Exposed=Window]
 interface KeyframeEffect : AnimationEffect {
-    constructor((Element or CSSPseudoElement)? target,
+    constructor(Element? target,
```

Comment:
Need to depend on dom.idl

Reply:
Looks like there's a pre-existed all-browser failure because of this too:
https://wpt.fyi/results/web-animations/interfaces/KeyframeEffect/idlharness.window.html?run_id=331320003&run_id=327490007&run_id=344960002&run_id=350400002